Sources/UnidirectionalBinding.swift

```diff
@@ -276,3 +277,62 @@ extension BindingTarget where Value: OptionalProtocol {
 		return target <~ property.producer
 	}
 }
+
+/// A pseudo `BindingTarget`, constructed from a closure and a lifetime.
+public final class AnyBindingTarget<Value>: BindingTarget {
+	public let lifetime: Lifetime
+	private let setter: (Value) -> Void
+
+	/// Wraps a `BindingTarget`.
+	///
+	/// - parameters:
+	///   - target: The binding target to wrap.
+	public init<T: BindingTarget>(_ target: T) where T.Value == Value {
+		lifetime = target.lifetime
+		setter = target.consume
```

Comment:
However, if I have a text field, and a binding target that sets its `text` property, retaining the binding target is not the same as retaining the text field. So I think you're right, the important thing is that the binding target doesn't affect the lifetime of the subject of that binding.